index.html

```diff
@@ -40,150 +50,167 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+                    <li class="skill_container"><span class="skill">Ruby</span></li>
+                </ul>
+                <button type="button" class="project_link"><spsn class="link_text">See project</span></button>
             </div>
-            <div class="project_image"></div>
-            <ul class="project_skills">
-                <li class="skill_container"><span class="skill">HTML</span></li>
-                <li class="skill_container"><span class="skill">Bootstrap</span></li>
-                <li class="skill_container"><span class="skill">Ruby</span></li>
-            </ul>
-            <button type="button" class="project_link"><span class="link_text">See project</span></button>
-        </div>
-        <div class="project_container_6">
-            <div class="project_title">
-                <h3>Trading Website</h3>
-            </div>
-            <div class="project_description">
-                <p>A daily selection of privately personalized reads; 
-                no accounts or sign-ups required. Has been the 
-                industry's standard.</p>
-            </div>
-            <div class="project_image"></div>
-            <ul class="project_skills">
-                <li class="skill_container"><span class="skill">HTML</span></li>
-                <li class="skill_container"><span class="skill">Bootstrap</span></li>
-                <li class="skill_container"><span class="skill">Ruby</span></li>
-            </ul>
-            <button type="button" class="project_link"><spsn class="link_text">See project</span></button>
-        </div>
+
+        </div>  
     </section>
     <section class="about_me">
-        <div class="bio_header">
-            <h1>About me</h1>
-        </div>
-        <div class="bio">
-            <p>Hello I'm a software developer! I can help you build a product , feature or website 
-            Look through some of my work and experience! If you like what you see and have a 
-            project you need coded, don't hesitate to contact me.</p>
+        <div class="about_me_container">
+            <div class="bio_header">
+                <h2>About me</h2>
+            </div>
+            <div class="bio">
+                <p>Hello I'm a software developer! I can help you build a product , feature or website 
+                Look through some of my work and experience! If you like what you see and have a 
+                project you need coded, don't hesitate to contact me.</p>
+            </div>
+    
+            <a href="#" class="resume_link">
+                <button type="button" class="resume_button">
+                    <span class="link_text">Get my resume</span>
+                </button>
+            </a>
         </div>
-
-        <a href="#" class="resume_link">
-            <button type="button" class="resume_button">
-                <span class="link_text">Get my resume</span>
-            </button>
-        </a>
         <div class="about_me_illustration"></div>
```

Comment:
- As mentioned earlier about the background, the background image should not have its `div` instead it should be used in the section or under the HTML tag. For example for this particular case, the background image should be attached to the `about-me-container` or `about-me` ☺️.